web/main/views.py

```diff
@@ -3076,19 +3076,22 @@ def internal_search(request):
     if author:
         filters["attribution"] = author
     if school:
-        filters["affiliation"] = school
+        filters["institution"] = school
 
     results, counts, facets = SearchIndex.search(
         category,
         page=page,
         query=query,
         filters=filters,
-        facet_fields=["attribution", "affiliation"],
+        facet_fields=["attribution", "institution"],
         order_by=request.GET.get("sort"),
     )
     full_counts = SearchIndex.counts(query=SearchIndex.objects.all())
 
-    results.from_capapi = False
```

Comment:
Enabling mypy on these functions flagged that this property wasn't on the object, and indeed I couldn't find the string in the code anywhere.

Reply:
Ah, I bet that is a relic from before we had `LegalDocument` and `LegalDocumentSource` classes.

Reply:
yeah https://github.com/harvard-lil/h2o/blob/315c8d4c7b6e314ec7ced886daca8cce33135821/web/search/views.py#L51